src/src/main.rs

```diff
@@ -555,9 +555,9 @@ fn allowance(owner: Principal, spender: Principal) -> Nat {
         .clone()
 }
 
-#[query(name = "getLogo")]
-#[candid_method(query, rename = "getLogo")]
-fn get_logo() -> String {
+#[query(name = "logo")]
```

Comment:
#[query] alone might be okay

Reply:
without `#[candid_method(query, rename = "logo")]`, cargo run will not generate candid

Reply:
sorry, I misunderstood, will update